### ISSUE TITLE
Crete rippleDelayClick

### ISSUE THREAD

**Issue by cosic** [traex/RippleEffect#53]
Add rv_rippleDelayClick attribute for waiting of finish animation for calling performClick() action

**Comment by cosic** [comment]
For example, on Android 2.3 i observed case when by click on button that invoke transfer on new activity I didn't see ripple animation. And only when I return on previous screen I see this animation.

**Comment by bbeckk** [comment]
is this attribute available? Since the ripple doesnt complete when another activity starts & remaining effect completes when back button is pressd. Its annoying

**Comment by cosic** [comment]
@bbeckk I created [pull reques](https://github.com/traex/RippleEffect/pull/54), but @traex still not took it. You can see realization of rv_rippleDelayClick at [my fork](https://github.com/cosic/RippleEffect/tree/am/%2353)

**Comment by anantshah93** [comment]
I had problem on activity button click added but after on back press it is showing animation.

**Comment by cosic** [comment]
@anantshah93 Did you try ripple:rv_rippleDelayClick="true"  from [my fork](https://github.com/cosic/RippleEffect/tree/am/%2353)?

**Comment by anantshah93** [comment]
@cosic yes, I tried ripple:rv_rippleDelayClick="true" but its showing error. i just add lib to gradle. I don't know to to use this by using your fork. can u explain me?

Thank you for your prompt reply.

**Comment by cosic** [comment]
@anantshah93 Yesterday I upgraded dependencies and fixed the error. Check it now.

**Comment by anantshah93** [comment]
@cosic  I tried 34 min ago at the time of comment and after confirm replied to your comment with error.
And again i tried just 2 min before but showing same error.

**Comment by cosic** [comment]
@anantshah93 I guess you are doing something wrong. It works. I cheched it now. Try to download code from the master branch.
